technische-universitat-dresden-medizin.csl

```diff
@@ -50,8 +50,8 @@
     </names>
   </macro>
   <macro name="author-short">
-    <names variable="author">
-      <name form="short" suffix="," and="text" delimiter-precedes-last="never" initialize-with=". "/>
+    <names variable="author" suffix=", ">
+      <name form="short" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize="false" sort-separator=""/>
```

Comment:
Delete ```initialize="false" sort-separator=""```` -- neither does anything here (initialise="false" is the default absent an initialize-with attribute and sort-separator is only used if name-as-sort-order is set to first or all).